docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -776,6 +778,43 @@ In this example, the table is partitioned by the month of ``order_date``, a hash
         country VARCHAR)
     WITH (partitioning = ARRAY['month(order_date)', 'bucket(account_number, 10)', 'country'])
 
+Sorted tables
+-------------
+
+Iceberg tables may have a specified list of sort columns. When writes are made to the table, each file will
+individually be sorted by these columns. Sorting can be particularly helpful when selective reads are frequently
+done which filter data using a column with a high cardinality.
+
+A table's sort order is provided using the ``sorted_by`` property. For example::
+
+    CREATE TABLE iceberg.testdb.customer_orders (
+        order_id BIGINT,
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR,
+        country VARCHAR)
+    WITH (sorted_by = ARRAY['order_date'])
+
+Sorting can be combined with partitioning on the same column, for example if a timestamp
+column is often queried selectively but the data is sparse enough that a higher granularity
+partition transform produces a better data layout::
+
+    CREATE TABLE iceberg.testdb.customer_orders (
+        order_id BIGINT,
+        order_date DATE,
+        account_number BIGINT,
+        customer VARCHAR,
+        country VARCHAR)
+    WITH (partitioning = ARRAY['month(order_date)'], sorted_by = ARRAY['order_date'])
+
+If you would like to disable sorting for a particular operation, you can do so by setting the
```

Comment:
You can disable sorted writing for with the session property ``sorted_writing_enabled`` set to ``false``.

However .. is that a catalog session property?